editor/src/core/webpack-loaders/json-loader.ts

```diff
@@ -0,0 +1,15 @@
+import { LoadModule, loadModuleResult, MatchFile, ModuleLoader } from './loader-types'
+
+const matchFile: MatchFile = (filename: string) => {
+  return ['.json'].some((extension) => filename.endsWith(extension))
+}
+
+const loadModule: LoadModule = (filename: string, contents: string) => {
+  const loadedContents = `module.exports = ${contents}`
+  return loadModuleResult(filename + '.js', loadedContents)
```

Comment:
This was for an issue in the ts-worker. All loaders append `.js`, because they are creating js files!

Reply:
are we still using the ts-worker for these modules? or did we at one point replace / simplify that? ugh my memory is very vague

Reply:
I've just had a cursory test of not appending the `.js` and adding `.json` [here](https://github.com/concrete-utopia/utopia/blob/master/editor/src/core/es-modules/evaluator/evaluator.ts#L109) and nothing visibly broke, but I'd rather someone spend the time to properly make sure it is no longer needed before committing to making that change.

The original work btw was here: https://github.com/concrete-utopia/utopia/pull/631 - I _think_ the .js was added because that was originally loading it into browserfs for the ts-worker, so it might be the case that it isn't needed.